script.js

```diff
@@ -8,7 +8,6 @@ var generateRandomInteger = function () {
   var randomDec = Math.random() * 3;
```

Comment:
Would be good to spell out the full variable name `randomDecimal`, in case whoever reads your code doesn't have the full context of what you are doing